net-libs/tox/files/initd

```diff
@@ -0,0 +1,30 @@
+#!/sbin/runscript
+
+PIDDIR=/run/tox-bootstrapd
+PIDFILE="${PIDDIR}"/tox-bootstrap.pid
+
+depend() {
+	need net
+}
+
+start() {
+	ebegin "Starting tox-dht-bootstrap daemon"
+
+	checkpath -d -q -o ${TOX_USER}:${TOX_GROUP} "${PIDDIR}"
```

Comment:
(minor) I'd quote `${TOX_USER}:${TOX_GROUP}` just in case. This will make errors more obvious if space accidentally sneaks in somewhere.